src/cli/argument-parser.js

```diff
@@ -104,6 +104,8 @@ export default class CLIArgumentParser {
             .option('--dev', 'enables mechanisms to log and diagnose errors')
             .option('--qr-code', 'outputs QR-code that repeats URLs used to connect the remote browsers')
             .option('--sf, --stop-on-first-fail', 'stop an entire test run if any test fails')
+            .option('--disable-test-syntax-validation', 'disable test syntax validation')
```

Comment:
\cc @VasilyStrelyaev for approval

Reply:
Well, we don't describe much here...
What if we say, for instance,
```js
            .option('--disable-test-syntax-validation', 'allows you to provide .js files that do not contain tests, while tests are imported or generated dynamically.')
```
(not checked with Dirk, just a suggestion)

Reply:
I agree with @VasilyStrelyaev. We can provide more detailed option description.
```js
disable `test` and `fixture` directives checking/validation (it allows to run dynamically loaded tests)
```

Reply:
I've commited with Vasily case for now, in any case we are waiting for Dirk

Reply:
Ok. Need to discuss with @VasilyStrelyaev

Reply:
Dirk's version:

`disables checks for 'test' and 'fixture' directives to run dynamically loaded tests`